### ISSUE TITLE
Exclude packages not matter the version

### ISSUE THREAD

**Issue by marianocodes** [davglass/license-checker#187]
Is there a way to exclude packages no matter the version? or exclude packages by name?
e.g
**Package Name:** `react@1.16.3` 
**Use:** `--excludePackages 'react'`

**Comment by burtyish** [comment]
I couldn't figure out why my excluded packages were included anyway until I saw this issue.
A feature like this would be very useful.

**Comment by getupp** [comment]
I noticed that you need to add the semicolon to the start and end of the list like this:
--excludePackages ';package1@0.0.1;package2@4.2.1;package3@0.0.2;'"

If not, only the package2 will be filtered.

**Comment by brettz9** [comment]
If the implementation details are similar, I'd also be interested in `--packages` checking packages within production and/or development regardless of version.

**Comment by RSeidelsohn** [comment]
You might want to have a look into my forked version of license-checker: [~~license-checker-dkbcodefactory~~](https://www.npmjs.com/package/license-checker-dkbcodefactory) [license-checker-rseidelsohn](https://www.npmjs.com/package/license-checker-rseidelsohn). I believe it now does just what you were asking for (I just yet didn't test it in combination with the `--production` and `--development` options.

**Comment by aalexgabi** [comment]
I did a very basic implementation here https://github.com/davglass/license-checker/pull/255/files
Not sure it's going to be accepted by maintainers though.

@RSeidelsohn Thanks for the fork. It seems to be a better implementation than mine :grin: 
The direct link for your forked package is now: https://www.npmjs.com/package/license-checker-rseidelsohn
Maybe you can update your comment.

**Comment by RSeidelsohn** [comment]
You are right, @aalexgabi - it took more than a year, but I finally saw your comment and updated mine. Thx!
